util/rom-ext-manifest-generator.py

```diff
@@ -92,23 +94,47 @@ def generate_defines(fields):
         required_alignment_bytes = required_alignment_bits // 8
 
         # The 8-byte two-step https://zinascii.com/2014/the-8-byte-two-step.html
-        # This ends up aligning `current_offset_bytes` to `required_alignment_bytes`
+        # This ends up aligning `new_offset_bytes` to `required_alignment_bytes`
         # that is greater than or equal to `current_offset_bytes`.
-        current_offset_bytes = (current_offset_bytes + required_alignment_bytes - 1) \
+        new_offset_bytes = (current_offset_bytes + required_alignment_bytes - 1) \
             & ~(required_alignment_bytes - 1)
 
+        if new_offset_bytes != current_offset_bytes and verbose:
+            print("0x{:04x} - 0x{:04x}: - (realignment) -".format(
+                current_offset_bytes, new_offset_bytes))
+
+        current_offset_bytes = new_offset_bytes
+        current_offset_alignment = current_offset_bytes & -current_offset_bytes
```

Comment:
Could you maybe in a comment drop an example with `4` and `-4` (...00000100 & ...11111100). It is pretty obvious what it does, but I didn't come across this pattern match.

Reply:
A better example is `6`, which should have output `2`.

Reply:
Sounds good.